websites/D/Discord Turkiye/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "nowongame",
+		"id": "686990422198845452"
+	},
+	"service": "Discord Turkiye",
+	"description": {
+		"en": "Discord Turkiye is the largest Forum/Discord community in Turkey, which contains updates, news, servers, bots, trades and many categories.",
+		"tr": "Discord Turkiye, güncellemeler, haberler, sunucular, botlar, ticaretler ve birçok kategoriyi içerisinde barındıran Turkiye'nin en büyük Forum/Discord topluluğudur."
```

Comment:
There's no plural of the word "ticaret" so the use of "ticaretler" is not correct. Just call it "ticaret", which will basically get you the same meaning.

Replace "Turkiye" with "Türkiye" in the following string:
> barındıran Turkiye'nin...

Make "Forum" lowercase, initial letter shouldn't be uppercase.